staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/admission.go

```diff
@@ -155,10 +155,6 @@ func (a *QuotaAdmission) ValidateInitialization() error {
 
 // Validate makes admission decisions while enforcing quota
 func (a *QuotaAdmission) Validate(ctx context.Context, attr admission.Attributes, o admission.ObjectInterfaces) (err error) {
-	// ignore all operations that correspond to sub-resource actions
-	if attr.GetSubresource() != "" {
-		return nil
-	}
```

Comment:
how bad this may be? Can code below have some assumptions about subresource already being checked and get broken by this change? Is there any perf implications? I wonder if we should only allow resize subresource here

Reply:
I don't know this logic well enough to reason about it. It just looks scary that we introduce a large change here without any safeguards in case we missed anything.

@liggitt @jpbetz who know this code best

Reply:
> It just looks scary that we introduce a large change here without any safeguards in case we missed anything.

Yeah, I had a similar concern and hence I added unit tests to avoid any side-effects.

Reply:
We will need to update e2e tests to get more confidence, I believe. Will it be a separate PR for this ?

Reply:
Oh, I saw it is already done in this PR ! Good work @AnishShah !